### PR TITLE
RavenDB-5342 & RavenDB-4608

### PR DESCRIPTION
**Pull request by ppekrol**

### CODE REVIEW

File: src/Voron/Data/Tables/Table.cs [hunk 1]
```diff
@@ -619,6 +619,32 @@ public IEnumerable<SeekResult> SeekForwardFrom(TableSchema.SchemaIndexDef index,
...
+        public long GetNumberEntriesFor(TableSchema.FixedSizeSchemaIndexDef index, long afterValue, out long totalCount)
+        {
+            var fst = GetFixedSizeTree(index);
+
+            totalCount = fst.NumberOfEntries;
+            if (afterValue == 0 || totalCount == 0)
+                return totalCount;
+
+            long count = 0;
+            using (var it = fst.Iterate())
```
Comment (ayende) [inline src/Voron/Data/Tables/Table.cs:631]: That can be extremely expensive, and it will require us to force load all the relevant data.

Comment (ayende) [reply src/Voron/Data/Tables/Table.cs:631]: That means that accessing the stats would be very slow and generate lots of I/O underload

Comment (ppekrol) [reply src/Voron/Data/Tables/Table.cs:631]: If I'm not wrong, only index will load, tested this on licensing and the memory usage was very low.

It takes around ~250ms (in Debug) to load those stats on a licensing database.

Comment (ayende) [reply src/Voron/Data/Tables/Table.cs:631]: My problem is that you are trying to get accurate answer, and on large dbs, that can be problematic.
Consider the case of Stackoverflow db we are testing, 45GB in size, that will add a lot of overhead, and we can't afford it in stats endpoint that we'll query when the system is already under load.

**Pull request merged**